docs/data/data-grid/api-object/api-object.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Data Grid - API object
+---
+
+# Data grid - API object
+
+<p class="description">Interact with the grid using its api.</p>
+
+## What is the API object ?
+
+The API object is an interface containing the state and all the methods needed to interact imperatively with the grid.
+
+You can find the list of all the available API methods on the [GridApi page](/x/api/data-grid/grid-api/).
+
+:::warning
+All the methods prefixed by `unstable_` are private.
```

Comment:
I thought `unstable_` was used more in the direction of an `alpha` version of an API method.
Would it make sense to consider `private_` in these cases?

Reply:
Right now it's the same prefix
And right now no `unstable_` is documented in a markdown file, even if theoretically we could indeed be used for features in alpha phase.

I suggest we leave it as is (with rewording if you want).
And we discuss if it's worth splitting in two prefixes in another PR.